qemu/tests/cfg/single_driver_install.cfg

```diff
@@ -21,6 +24,8 @@
     del devcon_dirname
     vio_driver_chk_cmd = 'driverquery /si | find /i "%s"'
     key_to_install_driver = "tab;kp_enter"
+    need_uninstall = yes
```

Comment:
I do not think this parameter is necessary as all sub case need un-install the old driver before install the new one.

How do you think about update script directly to make it as one must step for this case.